### ISSUE TITLE
Grid won't expand height after starting empty

### ISSUE THREAD

**Issue by galeano** [ceolter/ag-grid#498]
I'm using ag-grid to display results from a search. The grid needs to start empty until a search criteria is entered.

The problem is after data has been gathered from a service and injected into the grid using setRowData (although it behaves the same using datasource too), the container does not expand to fit the new data. It looks as though nothing happened.

I don't want to use a fixed pixel height because if I do, the grid won't expand either and stay with whatever setting it starts with.

Basically I need the grid to expand accordingly based on either a) the amount of rows I want to show per page or b) the total amount of rows in the data set.

Is this behavior by design or an issue? Do I have any alternatives?

Here's an example of the situation: https://jsfiddle.net/fg1984/s6dkw8e0/3/

Thanks.

**Comment by ceolter** [comment]
no you can't do that. the grid does not resize based on the content.

i plan putting this in q1 next year.

i'm closing this as it's a duplicate of this issue:
https://github.com/ceolter/ag-grid/issues/402

**Comment by galeano** [comment]
Ok, great. Thank you.

**Comment by rajgopalk** [comment]
[no you can't do that. the grid does not resize based on the content.

i plan putting this in q1 next year.

i'm closing this as it's a duplicate of this issue:
#402]

is that still pending?

**Comment by ceolter** [comment]
yes, this is still pending. hope to do it in the coming months, it's on my list of 'medium sized things to do' with about 10 other items.

**Comment by hardy1988** [comment]
Any update on this issue/feature ?
